backend/src/Github/Entity/GithubPullRequest.php

```diff
@@ -61,6 +61,12 @@ class GithubPullRequest
     #[ORM\Column(type: Types::JSON, nullable: false)]
     public array $diffFiles = [];
 
```

Comment:
BOT4:
The nullable annotation here conflicts with the specification in the migration file `backend/migrations/Version20240313081642.php` which indicates that the column `reviews` can be `NULL`. Ensure the annotation in other relevant places also reflects this to maintain consistency.